.github/workflows/ci.yml

```diff
@@ -190,7 +190,6 @@ jobs:
               generate-matrix \
               --kind acceptance-test \
               "$CODEGEN_TESTS_FLAG" \
-              --tags all xplatform_acceptance \
```

Comment:
I'd love tho see this tag disappear from the source files too, and we should also rename/merge the test files with `acceptance` un the file name.

With that, I think we can simplify the `get-job-matrix.py` and its usage in the actions setup.

This also makes me wonder if we are now running these tests twice in the merge queue?

Reply:
Or am I confused by "acceptance" vs "integration" test here?

Reply:
Yeah, we can probably drop the `xplatform_acceptance` tag completely. 

> This also makes me wonder if we are now running these tests twice in the merge queue?

No, we should only be running them once (well three times in total, once for each platform).  

This is now essentially the same as the integration test matrix above, so there is no longer a difference between integration and acceptance tests.

`--tags all xplatform_acceptance` means "include everything with the `all` tag, and exclude everything with the `xplatform_acceptance` tag (since it's set up like this: `//go:build (python || all) && !xplatform_acceptance`)".  Note that the `all` tag is added by default by `get-job-matrix.py generate-matrix`.